### PR TITLE
feat: Add prop to specify whether to display Runtime dropdown and labels

### PR DESCRIPTION
**Pull request by jaimereyesmx**
Allow the flexibility to specify whether Runtime dropdown and labels should be visible.

After this change the designer can be customized to not include runtime dropdown and labels.

"Add an action" panel
image

Search results
image

Triggers
image

### CODE REVIEW

File: libs/designer-ui/src/lib/panel/recommendationpanel/browseResults.tsx [hunk 1]
```diff
@@ -41,6 +42,7 @@ export const BrowseGrid = (props: BrowseGridProps) => {
               brandColor={brandColor}
               onClick={onConnectorSelected}
               category={getConnectorCategoryString(connector.id)}
+              displayRuntimeInfo={displayRuntimeInfo}
```
Comment (ccastrotrejo) [inline libs/designer-ui/src/lib/panel/recommendationpanel/browseResults.tsx:45]: Missing dependency in react hook

Comment (jaimereyesmx) [reply libs/designer-ui/src/lib/panel/recommendationpanel/browseResults.tsx:45]: Nice catch, thanks!

Comment (jaimereyesmx) [reply libs/designer-ui/src/lib/panel/recommendationpanel/browseResults.tsx:45]: Resolved it.

**Comment by ccastrotrejo** [comment]
Seems good to me. It seems that the default value for `displayRuntimeInfo` is true, however my concerns, and maybe I am not aware of this or didn't catch it in the PR, is in which scenarios when want it to be false and if we are going to implement that "switch" in vscode and portal  @hartra344

**Review by ccastrotrejo — APPROVED** [review]

**Review by hartra344 — APPROVED** [review]

**Pull request merged**